jmx-metrics/src/main/resources/target-systems/hbase.groovy

```diff
@@ -16,45 +16,45 @@
 
 def beanMasterServer = otel.mbeans("Hadoop:service=HBase,name=Master,sub=Server")
 otel.instrument(beanMasterServer, "hbase.master.region_server.count",
-  "The number of region servers.", "{servers}",
+  "The number of region servers.", "{server}",
```

Comment:
I'm probably misremembering, but I thought the plan was to not change things in the existing `jmx-metrics` component, and fix them only in the `jmx-scraper` implementation?

Reply:
That was our initial plan, however we (or at least I) changed a bit our mind about it for units because:
- units is just metadata, the values captured remain the same and it's quite unlikely any implementation strongly relies on those values (for example, it might break some UI/i18n, but the values remain the same). Changing the metric or its attribute names would not be the same minor impact.
- the [semconv conventions about this are stable](https://opentelemetry.io/docs/specs/semconv/general/metrics/#instrument-units), thus those changes will eventually have to be done at some point, and this makes one less difference to deal with when enhancing the mappings later.

We already did similar changes with other JMX scraper PRs, for example:
- https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1531
- https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1517

Reply:
I may only add that this change should be perfectly safe since the only changes were made in unit annotations that are discarded by the parsers anyway (according to [these docs](https://ucum.org/ucum#para-curly)).

Reply:
makes sense, thanks!